di-ipv-core-stub/src/main/java/uk/gov/di/ipv/stub/core/config/uatuser/Identity.java

```diff
@@ -1,10 +1,12 @@
 package uk.gov.di.ipv.stub.core.config.uatuser;
 
+import java.util.List;
+
 public record Identity(
         int rowNumber,
         String accountNumber,
         String ctdbDatabase,
-        UKAddress UKAddress,
+        List<UKAddress> UKAddress,
```

Comment:
rename to `addresses`. At present only UK addresses are supported therefore the UK prefix on the variable name doesn't really add any value